rtest.xml

```diff
@@ -2,10 +2,14 @@
 <test sets="psdb">
   <run name="all_tests">{CTEST_FILTER} {CTEST_REGEX}</run>
 </test>
 <test sets="osdb">
   <run name="all_tests">{CTEST_FILTER} {CTEST_REGEX}</run>
 </test>
+<test sets="smoketest">
+  <run name="smoke_tests">{SMOKE_TEST}</run>
+</test>
```

Comment:
Please rename test set as just "smoke" as that is the name convention set up by rocblas (and in documentation -t smoke)

Reply:
done!